diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
index 0a8820c..269adce 100644
--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
+++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
@@ -6289,6 +6289,10 @@ function Droppable(props) {
   var children = props.children,
       droppableId = props.droppableId,
       type = props.type,
+      //Fix for isDropDisabled must be a boolean
+      ignoreContainerClipping = props.ignoreContainerClipping,
+      //
       isDropDisabled = props.isDropDisabled,
       isCombineEnabled = props.isCombineEnabled,
       direction = props.direction,
@@ -6296,7 +6300,7 @@ function Droppable(props) {
       renderClone = props.renderClone;
   var contextId = useMemo(function () {
     return getContextId();
-  }, []);
+  }, [getContextId]);
   var ref = useRef(null);
   var placeholderRef = useRef(null);
   var droppable = useMemo(function () {
@@ -6323,7 +6327,9 @@ function Droppable(props) {
   var getDroppableRef = useMemo(function () {
     return function (element) {
       ref.current = element;
-    };
+      if(ref.current)
+        ref.current.setAttribute('data-rbd-droppable-id-2', droppableId);
+      };
   }, []);
   var getPlaceholderRef = useMemo(function () {
     return function (element) {
@@ -6341,12 +6347,15 @@ function Droppable(props) {
       placeholder: placeholder
     });
   }, [children, isDropDisabled, placeholder]);
+  //Fix for isDropDisabled must be a boolean
+  var memoizedIsDropDisabled = useMemo(function () { return isDropDisabled; }, [isDropDisabled]);
+  //
   useEffect(function () {
     if (isDropDisabled) {
       if (placeholder) {
         warnOnce("Droppable setup issue: you have provided a placeholder but have specified that the droppable is disabled. A placeholder will not be mounted for a disabled droppable. This is not a critical error, but it may be a sign that you have a setup problem");
       }
-    }
+    } //Fix for isDropDisabled must be a boolean
   }, [isDropDisabled, placeholder]);
   var onChildrenChange = useValidation(props, droppable, ref);
   onChildrenChange();
@@ -6355,7 +6364,9 @@ function Droppable(props) {
     return children(provided, snapshot);
   }, [children, provided, snapshot]);
   useDroppablePublisher(droppable);
-  return content;
+  //Fix for isDropDisabled must be a boolean
+  return /*#__PURE__*/createElement(Fragment, null, content);
+  //
 }
 
 var getMaxScroll = function getMaxScroll(_ref) {
diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js
index b29b14b..e8251e6 100644
--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js
+++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js
@@ -6681,6 +6681,10 @@ function Droppable(props) {
   var children = props.children,
       droppableId = props.droppableId,
       type = props.type,
+      //Fix for isDropDisabled must be a boolean
+      ignoreContainerClipping = props.ignoreContainerClipping,
+      //
       isDropDisabled = props.isDropDisabled,
       isCombineEnabled = props.isCombineEnabled,
       direction = props.direction,
@@ -6688,7 +6692,7 @@ function Droppable(props) {
       renderClone = props.renderClone;
   var contextId = react.useMemo(function () {
     return getContextId();
-  }, []);
+  }, [getContextId]);
   var ref = react.useRef(null);
   var placeholderRef = react.useRef(null);
   var droppable = react.useMemo(function () {
@@ -6715,7 +6719,9 @@ function Droppable(props) {
   var getDroppableRef = react.useMemo(function () {
     return function (element) {
       ref.current = element;
-    };
+      if(ref.current)
+        ref.current.setAttribute('data-rbd-droppable-id-2', droppableId);
+      };
   }, []);
   var getPlaceholderRef = react.useMemo(function () {
     return function (element) {
@@ -6733,12 +6739,15 @@ function Droppable(props) {
       placeholder: placeholder
     });
   }, [children, isDropDisabled, placeholder]);
+  //Fix for isDropDisabled must be a boolean
+  var memoizedIsDropDisabled = react.useMemo(function () { return isDropDisabled; }, [isDropDisabled]);
+  //
   react.useEffect(function () {
     if (isDropDisabled) {
       if (placeholder) {
         warnOnce("Droppable setup issue: you have provided a placeholder but have specified that the droppable is disabled. A placeholder will not be mounted for a disabled droppable. This is not a critical error, but it may be a sign that you have a setup problem");
       }
-    }
+    } //Fix for isDropDisabled must be a boolean
   }, [isDropDisabled, placeholder]);
   var onChildrenChange = useValidation(props, droppable, ref);
   onChildrenChange();
@@ -6747,7 +6756,9 @@ function Droppable(props) {
     return children(provided, snapshot);
   }, [children, provided, snapshot]);
   useDroppablePublisher(droppable);
-  return content;
+  //Fix for isDropDisabled must be a boolean
+  return /*#__PURE__*/react.createElement(react.Fragment, null, content);
+  //
 }
 
 var getMaxScroll = function getMaxScroll(_ref) {

